src/wp-admin/includes/misc.php

```diff
@@ -352,7 +352,7 @@ function update_recently_edited( $file ) {
 		$oldfiles[] = $file;
 	}
 
-	update_option( 'recently_edited', $oldfiles );
+	update_option( 'recently_edited', $oldfiles, false );
```

Comment:
This still needs to be reverted, see https://github.com/WordPress/wordpress-develop/pull/6465#discussion_r1731876165.
```suggestion
	update_option( 'recently_edited', $oldfiles );
```